data/features/m/m.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Description: Function to search arguments in the manual of a command
+# Argument 1: String representing a system binary (example: ls)
+# Argument 2 (optional): String representing argument option of the binary (example: -A)
+m() {
+  if ! command -v man &>/dev/null; then
+    echo "Error: 'man' command not found."
+    return 1
+  fi
+
+  if [ $# -eq 0 ]; then # No arguments given
+    echo "Error: Invalid arguments."
+    return 1
+  fi
+
+  if [ $# -eq 1 ]; then
+    # Matches any line that starts with one or more spaces followed by a dash (-), one letter (lowercase or uppercase),
+    # a comma, a space, two hyphens (--), and one or more letters (lowercase or uppercase) or dashes (-).
+    # This pattern is used to match the short and long options that are documented in the manual page.
+    man_output="$(man "$@" | grep -E '^ +-[a-zA-Z], --[a-zA-Z-]+')"
```

Comment:
this regexp does nothing. You just need to check the manual

Reply:
Without it you also get the manual of the command, which includes arguments.

Reply:
okay got it